test/connection-pool.js

```diff
@@ -25,9 +25,6 @@ var proxyquire = require('proxyquire');
 var uuid = require('uuid');
 var util = require('util');
 
-var PKG = require('../package.json');
-var v1 = require('../src/v1');
-
 var fakeUtil = extend({}, common.util);
 var fakeUuid = extend({}, uuid);
 var fakeGrpc = extend({}, grpc);
```

Comment:
We can remove the stuff about `fakeGrpc` now that we aren't using it.

Reply:
Removed